src/configauditreports/page.scss

```diff
@@ -0,0 +1,34 @@
+.ConfigAuditReports {
+
+  .Badge {
+    &.severity-CRITICAL {
+      color: white;
+      background-color: #cc1814;
+    }
+
+    &.severity-HIGH {
+      color: white;
+      background-color: #ffa500;
+    }
+
+    &.severity-MEDIUM {
+      color: white;
+      background-color: #f0c20c;
+    }
+
+    &.severity-LOW {
+      color: white;
+      background-color: #096ab0;
+    }
+
+    &.severity-NEGLIGIBLE {
+      color: white;
+      background-color: #00b09b;
+    }
+
+    &.severity-UNKNOWN {
+      color: white;
+      background-color: #515456;
+    }
```

Comment:
For configuration audits we only have Critical, High, Medium, and Low severities.